CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.3.1
```

Comment:
@haoliu-amp I remember I had to separate the version number from the commit, like in Amplitude/unity-plugin and also like this [previous commit](https://github.com/amplitude/Amplitude-Flutter/commit/b0b264dddaba7a72300fa57c26cf7c056c9bb5c9). Are we enforcing this here?